include/seqan3/alphabet/aminoacid/all.hpp

```diff
@@ -0,0 +1,111 @@
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+
+/*!\file alphabet/aminoacid/all.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Meta-header for the aminoacid submodule; includes all headers from alphabet/aminoacid/.
+ */
+
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+
+/*!\defgroup aminoacid
+ * \brief Contains the amino acid alphabet, the reduced amino acid alphabet \TODO and translation from nucleotide
+ * into amino acid
+ * \ingroup alphabet
+ *
+ * \par Introduction
+ * Amino acid sequences are an important part of bioinformatic data processing and used by many applications
+ * and while it is possible to represent them in a regular std::string, it makes sense to have specialised data
+ * structures in most cases. This sub-module offers the 27 letter aminoacid alphabet as well as a reduced version
+ * that can be used with regular container and ranges.
+ *
+ * | Letter   | Description     |
+ * |:--------:|-----------------|
+ * |   A      | Alanine         |
+ * |   C      | Cysteine        |
+ * |   D      | Aspartic acid   |
+ * |   E      | Glutamic acid   |
+ * |   F      | Phenylalanine   |
+ * |   G      | Glycine         |
+ * |   H      | Histidine       |
+ * |   I      | Isoleucine      |
+ * |   K      | Lysine          |
+ * |   L      | Leucine         |
+ * |   M      | Methionine      |
+ * |   N      | Asparagine      |
+ * |   O      | Pyrrolysine     |
+ * |   P      | Proline         |
+ * |   Q      | Glutamine       |
+ * |   R      | Arginine        |
+ * |   S      | Serine          |
+ * |   T      | Threonine       |
+ * |   U      | Selenocysteine  |
+ * |   V      | Valine          |
+ * |   W      | Tryptophan      |
+ * |   Y      | Tyrosine        |
+ * |   B      | N or D          |
+ * |   J      | I or L          |
+ * |   Z      | Q or E          |
+ * |   *      | Terminator      |
+ * |   X      | any aminoacid   |
```

Comment:
I would also move O and U down to the lower block, because they are also not part of the canonical 20 amino acids.